manifest.json

```
@@ -1,7 +1,7 @@
 {
   "vendor": "vtex",
   "name": "store-link",
-  "version": "0.7.2",
+  "version": "0.8.0",
```

Our CI/CD will bump the app version
```suggestion
  "version": "0.7.2",
```